src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +36,11 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
+                .map(u -> u.age)
+                .filter(a -> a >= 30)
+                .filter(a -> a <= 45)
```

Comment:
```suggestion
                .filter(a -> a >= 30 && a <= 45)
```
와 같이 표현하면 filter 연산자를 한번만 거치고도 처리가 가능할 것 같습니다!